src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final String randomColor = new ColorSupplier().getRandomColor();
+    private final int randomFigure = random.nextInt(5);
+
+    public Figure getRandomFigure() {
+        switch (randomFigure) {
+            case 0:
+                return new Square(randomColor, random.nextInt(10));
+            case 1:
+                return new Rectangle(randomColor, random.nextInt(10), random.nextInt(10));
+            case 2:
+                return new RightTriangle(randomColor, random.nextInt(10), random.nextInt(10));
+            case 3:
+                return new Circle(randomColor, random.nextInt(10));
+            default:
+                return new IsoscelesTrapezoid(randomColor, random.nextInt(10),
+                        random.nextInt(10), random.nextInt(10));
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle("white",10);
```

Comment:
```suggestion
        return new Circle("white", 10);
```

Reply:
Still exists

Reply:
You have enum `Color`, why do use literal `String`? try to avoid magic numbers (e.g. 10)